inventories/vagrant.yaml

```diff
@@ -1,11 +0,0 @@
-[systems]
```

Comment:
The inventory is now generated by Vagrant. See the `ansible.groups` section of the Vagrantfile for details.